### ISSUE TITLE
AVT 3 - React Select issues with VoiceOver on macOS and iOS

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3338]
## Environment
~macOS Mojave version 10.14.5~
~Chrome Version 75.0.3770.100 (Official Build) (64-bit)~
~Carbon v10 - React~
~Voice Over~

Also tested on 
iOS 12.3.1 with Safari and Voice Over

## Detailed Description
For macOS:
Start VoiceOver 
Navigate to the Select box, VO does not announce the text in the select box.
Note: This works as expected in Windows  10 with Firefox.

For iOS:
Start VoiceOver
Swipe right to navigate to the select box. VoiceOver announces "Choose an option pop-up button selected." as expected. 
Double tap to open the menu, VoiceOver announces, "Choose an option pop-up button selected." as expected.
Swipe right again and nothing happens. 
Issue: Swipe right should navigate down the menu. In order to get focus on the list of options, the user must swipe left first before swiping right which is an unexpected behavior.

**Comment by aledavila** [comment]
@snidersd Is this still an issue for macOS on chrome ? I can't reproduce it.

**Comment by snidersd** [comment]
I retested on macOS and it reads correctly. I also retested on iOS 13.1.3 and there are still issues:
Swipe right, VO announces "Select main landmark"
Swipe right, VO announces "Optional helper text"
Swipe right to navigate to the select box. VoiceOver announces "Select. Choose an option pop-up button, double tap to activate the picker." 
Double tap to open the menu, VoiceOver does not announce anything. 
Expected Result: VO should announce that it is on a menu and the label of the option that has focus. 
Note: The menu focus is not consistent. The first couple of times it opened it was on "Choose an Option" and that option was checked (see screenshot). On reload and attempting it again there was no focus at all. 
<img width="817" alt="Screen Shot 2019-10-22 at 9 06 49 AM" src="https://user-images.githubusercontent.com/21676914/67294330-50946c00-f4ab-11e9-8f39-40b2d902ce4d.png">

**Comment by aledavila** [comment]
closing opened a separate issue with new bug
